api/src/identity-access-management/domain/errors.js

```diff
@@ -70,6 +70,12 @@ class PasswordResetDemandNotFoundError extends DomainError {
   }
 }
 
+class PixAdminLoginFromPasswordIsDisabledError extends DomainError {
```

Comment:
Utiliser un nom plus concis pour l’erreur (normalement on ne met pas de verbe dans le nom, par exemple `NotFoundError`, etc.) :
```suggestion
class PixAdminLoginFromPasswordDisabledError extends DomainError {
```